vulyk/utils.py

```diff
@@ -86,3 +87,11 @@ def get_tb():
     :return: traceback of the most recent exception
     """
     return sys.exc_info()[2]
+
+
+def get_template_path(app, name):
+    for x in iter(app.jinja_loader.list_templates()):
```

Comment:
What is the purpose of `iter` here? We do iterate over an already computed lists without any `sentinel` passed, don't we?
What wil happen to it in case if `app.config['TEMPLATE_BASE_FOLDERS']` is None or non-iterable object?